### PR TITLE
Input prometheus, use the same timestamp per call if no time is provided

### PR DESCRIPTION
**Pull request by aarnaud**
avoid the chance to have different timestamp during iteration of each
metric for one http.Response

This PR, fix an issue we discover. some time during the iteration of metrics, the timestamp change to pass to the next second and our metrics aren't align.

This only applied if TimestampMs is not provided in the prometheus metrics

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/prometheus/parser.go [hunk 1]
```diff
@@ -193,6 +195,8 @@ func Parse(buf []byte, header http.Header) ([]telegraf.Metric, error) {
 		}
 	}
 
+	// save the timestamp before iterate to avoid the change to have one second between 2 slice of metrics
```
[the remark this answers is not in the document]
Comment (aarnaud) [reply plugins/inputs/prometheus/parser.go]: Change applied

File: plugins/inputs/prometheus/parser.go [hunk 2]
```diff
@@ -55,6 +55,8 @@ func ParseV2(buf []byte, header http.Header) ([]telegraf.Metric, error) {
...
```
[the remark this answers is not in the document]
Comment (aarnaud) [reply plugins/inputs/prometheus/parser.go]: Change applied

**Comment by aarnaud** [comment]
Do you now when the next release is plan ?

Can I made a PR to have a bugfix on 1.12 or 1.13 soon ?

**Comment by ssoroka** [comment]
As soon as it's merged we'll back port it to 1.13.x (probably 1.13.4). Just waiting for @danielnelson to approve.

**Comment by aarnaud** [comment]
thanks @danielnelson for quick release. It's appreciate  
I can't find the 1.13.4 release on dockerhub : https://hub.docker.com/_/telegraf?tab=tags
Is there a delay, or an PR a can make to trigger a build ?

**Comment by danielnelson** [comment]
Sorry about this, currently it takes us a bit of time to get the official containers updated and unfortunately it isn't an automatic process.  There are two places you can keep an eye on for progress updates:
- First there is PR on https://github.com/influxdata/influxdata-docker/pull/378
- After this is merged a PR will be opened with https://github.com/docker-library/official-images/pull

**Pull request merged**